pkg/segment/search/searchaggs.go

```diff
@@ -1037,6 +1037,10 @@ func applySegmentStatsUsingDictEncoding(mcr *segread.MultiColSegmentReader, filt
 					}
 				}
 
+				if rawVal == nil {
+					continue
+				}
```

Comment:
In what cases does this run? Is this going to cause problems for certain aggregations like `count`? Or would that already be accounted for by some other flow for a non-DE column?